website/docs/setup/repos.md

```diff
@@ -159,3 +159,95 @@ need to use the following flag with the initial Corso `repo init` command:
 Corso also supports the use of object storage systems with no TLS certificate or with self-signed
 TLS certificates with the `--disable-tls` or `--disable-tls-verification` flags.
 [These flags](../../cli/corso-repo-init-s3) should only be used for testing.
+
+## Filesystem Storage
+
+Corso supports creating a repository on local or network attached filesystem.
+
+:::note
+Filesystem repositories aren't S3 compatible.
+:::
+
+### Filesystem storage prerequisites
+
+None.
+
```

Comment:
Can this section be omitted?
```suggestion
```

Reply:
Fair. I added it so that it's clear to the reader that there are no prerequisites. Compared to a perceived missing section.